src/app/features/tech-record/components/tech-record-summary/tech-record-summary.component.ts

```diff
@@ -95,8 +97,42 @@ export class TechRecordSummaryComponent implements OnInit {
     this.store.dispatch(updateEditingTechRecord({ techRecord: this.vehicleTechRecordCalculated }));
   }
 
+  removeAxle(axleEvent: any): void {
+    this.vehicleTechRecordCalculated = cloneDeep(this.vehicleTechRecordCalculated);
+
+    const axleToRemove = this.findAxleToRemove(axleEvent.axles);
+
+    this.vehicleTechRecordCalculated.axles = this.vehicleTechRecordCalculated.axles.filter(ax => {
```

Comment:
Hugely optional:
Here you could remove the axel already on say line 104 
then you could do a map on the axel to decrement the axel number on a splice of the array from axel to remove to end of the array

Reply:
Please no more on this function :')